capstone/capdb/migrations/0016_merge_20171214_1624.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-12-14 16:24
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('capdb', '0015_auto_20171212_1749'),
+        ('capdb', '0013_auto_20171213_1548'),
+    ]
+
+    operations = [
```

Comment:
what's the need for this migration?

Reply:
this is *not* in fact the 0013 migration from your pull request, so that makes total sense now